composer.json

```
@@ -8,7 +8,7 @@
         "php": "^8.0",
         "laravel/framework": "^9.0",
         "digitallyhappy/assets": "dev-bassets as 2.99.99",
-        "backpack/crud": "dev-coreuiv4 as 5.99.99",
+        "backpack/crud": "dev-tabler as 5.99.99",
         "backpack/pro": "dev-coreuiv4 as 1.99.99",
```

Let's use the `v6` branch of PRO, since [that one now has both Bootstrap v4 and Bootstrap v5 support](https://github.com/Laravel-Backpack/PRO/pull/131#issuecomment-1471665980):

```suggestion
        "backpack/pro": "v6.x-dev as 1.99.99",
```